libethereum/Transaction.cpp

```diff
@@ -73,6 +73,47 @@ TransactionException dev::eth::toTransactionException(Exception const& _e)
 	return TransactionException::Unknown;
 }
 
+string dev::eth::transactionExceptionToErrorMessage(Exception const& _e)
```

Comment:
These error messages are very RPC-specific, I would leave this function in `libweb3jsonrpc`, probably in `Eth.cpp`